javascript/javascript1/week4/homework/voice-assistant.js

```diff
@@ -0,0 +1,163 @@
+    command.toLowerCase().includes("from my todo")
+  ) {
+    let index = toDoList.indexOf(activity);
+    if (index > -1) {
+      toDoList.splice(index, 1);
+      return `Removed ${activity} from your todo`;
+    } else {
+      return `There is no such activity in your todo`;
+    }
+  }
+
+  //show full list of to-dos
+  if (command.toLowerCase().includes("on my todo")) {
+    if (toDoList.length === 1) {
+      return `There is only 1 activity in your to-do list: ${toDoList[0]}`;
+    } else if (toDoList.length > 1) {
+      return `You have ${toDoList.length} activities in your to-do list. There are: ${toDoList}`;
+    } else {
+      return `You don't have any activities in your to-do list. Please, add some first`;
+    }
+  }
+
+  //show current date
+  let today = new Date();
+  let date = today.getDate();
+  let month = monthes[today.getMonth()];
+  let year = today.getFullYear();
+  if (
+    command.toLowerCase().includes("day") &&
+    command.toLowerCase().includes("today")
+  ) {
+    return `Today is ${date}. of ${month} ${year}`;
+  }
+
+  //simple math equation
+  if (
+    command.toLowerCase().includes("+") ||
+    command.toLowerCase().includes("-") ||
+    command.toLowerCase().includes("*") ||
+    command.toLowerCase().includes("/")
+  ) {
+    let expression = commandSplit.slice(-3);
+    if (command.toLowerCase().includes("+")) {
+      return parseInt(expression[0]) + parseInt(expression[2]);
+    }
+    if (command.toLowerCase().includes("*")) {
+      return parseInt(expression[0]) * parseInt(expression[2]);
+    }
+    if (command.toLowerCase().includes("-")) {
+      return parseInt(expression[0]) - parseInt(expression[2]);
+    }
+    if (command.toLowerCase().includes("/")) {
+      return parseInt(expression[0]) / parseInt(expression[2]);
+    }
+  }
+
+  //set a timer (extra task: possibility to set timer either in minutes or in seconds)
+  if (
+    command.toLowerCase().includes("set") &&
+    command.toLowerCase().includes("timer")
+  ) {
+    let timerStr = commandSplit.slice(4);
+    console.log(`Timer set for ${timerStr.join(" ")}`);
+    let timeInMilSec;
+    if (timerStr[1] == "minutes" || timerStr[1] == "minute") {
+      timeInMilSec = parseInt(timerStr[0]) * 60 * 1000;
+    } else if (timerStr[1] == "seconds" || timerStr[1] == "second") {
+      timeInMilSec = parseInt(timerStr[0]) * 1000;
+    }
+    setTimeout(() => {
+      console.log("Timer done");
+    }, timeInMilSec);
+  }
+
+  /* extra task */
+  /* in a real situation this step should be right after greeting and introducing yourself */
+  if (command.toLowerCase().includes("birthday")) {
+    let birthday = commandSplit.slice(4);
+    personBirthday.push(parseInt(birthday[0]));
+    for (i = 0; i <= monthes.length; i++) {
+      if (birthday[2] == monthes[i]) {
+        personBirthday.push(monthes.indexOf(monthes[i]) + 1);
+      }
+    }
+  }
+
+  if (personBirthday[0] == date && personBirthday[1] == today.getMonth() + 1) {
```

Comment:
`===`

Reply:
Thanks for the review☺️ Actually, equality operators and assigning variables were confused me each time during homework. I really should learn more about it😅
Also I wanted to thank you for whole module. You've done a great job learning us all basics of JS which I sure is the hardest part. I learned a lot😊

Reply:
Thanks for the kind words :)